<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoteWindow</class>
 <widget class="QMainWindow" name="NoteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>889</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>889</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>784</x>
      <y>392</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>684</x>
      <y>392</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="keyLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select the Key:</string>
    </property>
   </widget>
   <widget class="QLabel" name="modeLabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select the Scale Type:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="keyComboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="modeComboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="secondsLabel">
    <property name="geometry">
     <rect>
      <x>587</x>
      <y>77</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Seconds:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondsInput">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>0 for Infinite Seconds</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>180</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removePushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="scaleTableWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>781</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
